service/pager/query_pager.hh

```diff
@@ -70,6 +70,11 @@ namespace pager {
  * is done even though it is.
  */
 class query_pager {
+public:
+    struct stats {
+        size_t rows_read_total = 0;
```

Comment:
Please add a comment explaining this statistic. In particular, I find myself puzzled what does "total" in the name of this metric mean? Is it a total of many pages? Probably not. If it's just one page, why don't we just call it "rows_read", and that's it?

Reply:
Oh but it is, that's why I appended `total` and hoped it's enough to explain that it gathers a total number of fetched rows for all pages that this pager fetches - in `fetch_page` you can see that this metrics accumulates the results:
~~~
_stats.rows_read_total += *qr.query_result->row_count();
~~~
. I'll add a comment that states it clearly.

Reply:
done